simple_database/dbmodule.py

```diff
@@ -0,0 +1,124 @@
+import json, os, pickle
+try:
+    from exceptions import *
+    from config import BASE_DB_FILE_PATH
+except:
+    from .exceptions import *
+    from .config import BASE_DB_FILE_PATH
+
+
+def initDbIndex(): #This function takes care of creating the index file, if it doesn't exist
+    if not os.path.exists(BASE_DB_FILE_PATH):
+        os.makedirs(BASE_DB_FILE_PATH)
+    if not os.path.isfile(os.path.join(BASE_DB_FILE_PATH, "path_index.json")):
+        open(os.path.join(BASE_DB_FILE_PATH, "path_index.json"), "a").close()
+        
+def create_database(db_name):
+    initDbIndex() #Check if the index file exists, or take care of it
+    if Db.doesDbExist(db_name):
+        raise ValidationError('Database with name "{}" already exists.'.format(db_name))
+    else:#If the db doesn't exist
+        db = Db(db_name)#Create it
+        with open(os.path.join(BASE_DB_FILE_PATH, "path_index.json"), "r") as file: #And update the index file
+            temp_file = file.read()
+            if temp_file: temp_file = json.loads(temp_file)
+            else: temp_file = {}
+            temp_file[db_name] = db.path
+        with open(os.path.join(BASE_DB_FILE_PATH, "path_index.json"), "w") as file:
+            file.write(json.dumps(temp_file))
+        return db
+        
+    # raise NotImplementedError()
+
+def connect_database(db_name):
```

Comment:
Your connect and create are very close.  Could they have been merged into one function, with perhaps a default parameter over-ridden for a connect?